docs/sources/reference/builder.md

```diff
@@ -343,6 +346,27 @@ accessible from the host by default. To expose ports to the host, at runtime,
 [use the `-p` flag](/userguide/dockerlinks) or
 [the -P flag](/reference/run/#expose-incoming-ports).
 
+## UNEXPOSE
+
+    UNEXPOSE <port> [<port>...]
+
+The `UNEXPOSE` instruction informs Docker that the container no longer wants
+to listen on the specified network ports at runtime.
+
+For example, given a image named `debug-app` defined with the following
+`Dockerfile`:
+
+    # docker build -t debug-app .
+    FROM debian:wheezy
+    EXPOSE 3000 8080
+
+We want to create a new image which exposes only port 3000. The
```

Comment:
As per the style guide, "we" refers to Docker and "you" refers to the user. Please correct.